website/docs/automation/gitops/argocd/app-of-apps/cleanup.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Suggest moving this page in to the app of apps section itself as part of setting up